PWGJE/Tasks/recoilJets.cxx

```diff
@@ -538,4 +538,4 @@ struct jetHadronRecoil_OO {
   }
 };
 
-WorkflowSpec defineDataProcessing(ConfigContext const& cfgc) { return WorkflowSpec{adaptAnalysisTask<jetHadronRecoil_OO>(cfgc, TaskName{"jet-hadron-recoil-oo"})}; }
+WorkflowSpec defineDataProcessing(ConfigContext const& cfgc) { return WorkflowSpec{adaptAnalysisTask<recoilJets>(cfgc, TaskName{"trigger-hadron-recoil-jet"})}; }
```

Comment:
There is no reason to use `TaskName`.

Reply:
Not addressed

Reply:
Could you clarify a bit more what you suggest?

Reply:
It is all written in the linter warnings. Which message is not clear?

Reply:
I see your point. So, the part ``TaskName{"recoil-jets"}`` can be omitted. It will be implemented in the next PR, thank you

Reply:
Yes, not only it can be omitted but it should be omitted. Please also fix the `struct` name to `RecoilJets` together with it. Thanks!